### PR TITLE
google-java-format

### LINKED ISSUE #126: Enforce code formatting
## I expect

Java code to have a standard formatting

## Instead

Every PR uses its own code formatting tool

## Notes

We can use https://github.com/google/google-java-format
and add it to the CI.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Fixes  #126. Code formatting

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Google Java Format
+      uses: axel-op/googlejavaformat-action@v3.2.0
```
Comment (ioggstream) [inline .github/workflows/lint.yml:26]:
```suggestion
      uses: axel-op/googlejavaformat-action@v3.1.0
```

**Comment by ioggstream** [comment]
@TheAntimist PTAL and if ok I'll rebase and merge.

**Review by TheAntimist — APPROVED** [review]
Somehow my notifications on Github, got lost in the noise. Anyway, these look good to me!

**Comment by TheAntimist** [comment]
@ioggstream Let me know if I can delete the branch? Or feel free to go ahead and do so.

**Comment by TheAntimist** [comment]
I just realized that you wanted to rebase all commits into one. Oh well, too late.

**Comment by ioggstream** [comment]
@TheAntimist let's try the `revert` feature :D

**Pull request merged**